lib/vsc/utils/fancylogger.py

```diff
@@ -85,6 +85,13 @@
 import weakref
 from distutils.version import LooseVersion
 
+try:
```

Comment:
can you implement this like the `mpiy4py` check below? (i.e. set it to false by default, and allow an environment variable to control the import)